core/src/test/java/org/elasticsearch/index/seqno/GlobalCheckpointTests.java

```diff
@@ -41,89 +49,77 @@
     public void setUp() throws Exception {
         super.setUp();
         checkpointService = new GlobalCheckpointService(new ShardId("test", "_na_", 0),
-            IndexSettingsModule.newIndexSettings("test", Settings.EMPTY), SequenceNumbersService.UNASSIGNED_SEQ_NO);
+            IndexSettingsModule.newIndexSettings("test", Settings.EMPTY), UNASSIGNED_SEQ_NO);
     }
 
     public void testEmptyShards() {
         assertFalse("checkpoint shouldn't be updated when the are no active shards", checkpointService.updateCheckpointOnPrimary());
-        assertThat(checkpointService.getCheckpoint(), equalTo(SequenceNumbersService.UNASSIGNED_SEQ_NO));
+        assertThat(checkpointService.getCheckpoint(), equalTo(UNASSIGNED_SEQ_NO));
     }
 
-    public void testGlobalCheckpointUpdate() {
+    private final AtomicInteger aIdGenerator = new AtomicInteger();
+    private Map<String, Long> randomAllocations(int min, int max) {
         Map<String, Long> allocations = new HashMap<>();
-        Set<String> active = new HashSet<>();
-        Set<String> insync = new HashSet<>();
-        Set<String> tracking = new HashSet<>();
-        long maxLocalCheckpoint = Long.MAX_VALUE;
-        for (int i = randomIntBetween(3, 10); i > 0; i--) {
-            String id = "id_" + i + "_" + randomAsciiOfLength(5);
-            long localCheckpoint = randomInt(200);
-            switch (randomInt(2)) {
-                case 0:
-                active.add(id);
-                maxLocalCheckpoint = Math.min(maxLocalCheckpoint, localCheckpoint);
-                    break;
-                case 1:
-                    insync.add(id);
-                    maxLocalCheckpoint = Math.min(maxLocalCheckpoint, localCheckpoint);
-                    break;
-                case 2:
-                    tracking.add(id);
-                    break;
-                default:
-                    throw new IllegalStateException("you messed up your numbers, didn't you?");
-            }
-            allocations.put(id, localCheckpoint);
+        for (int i = randomIntBetween(min, max); i > 0; i--) {
+            allocations.put("id_" + aIdGenerator.incrementAndGet(), (long)randomInt(1000));
         }
+        return allocations;
+    }
 
-        if (maxLocalCheckpoint == Long.MAX_VALUE) {
+    public void testGlobalCheckpointUpdate() {
+        Map<String, Long> allocations = new HashMap<>();
+        Map<String, Long> tmpAllocations = randomAllocations(0, 5);
```

Comment:
Can we call this `tmpActive` and not reuse it for the initializing as well `tmpInitializing`?